atom/browser/atom_browser_main_parts.cc

```diff
@@ -208,50 +208,6 @@ int AtomBrowserMainParts::PreCreateThreads() {
   // any callbacks, I just want to initialize the mechanism.)
   SecKeychainAddCallback(&KeychainCallback, 0, NULL);
 #endif  // defined(OS_MACOSX)
-  auto feature_list = base::FeatureList::GetInstance();
```

Comment:
why were these removed? This would cause crashes and break important functionality